pom.xml

```diff
@@ -1945,8 +1944,10 @@
             </dependency>
             <dependency>
                 <groupId>org.apache.logging.log4j</groupId>
-                <artifactId>log4j-api</artifactId>
-                <version>${log4j2.version}</version>
+                <artifactId>log4j-bom</artifactId>
+                <version>2.20.0</version>
```

Comment:
We can still use the ${log4j2.version} property instead of 2.20.0

Reply:
> We can still use the ${log4j2.version} property instead of 2.20.0

yes, you can do that but I didn't see the point in having a property only to use it one place.
plus it helps ensure that everything can _only_ be getting the version from `log4j-bom`.